### PR TITLE
Button Click Animation

### PR DESCRIPTION
**Pull request by elibosley**
# PR for #341.

## PR Status
**READY**

## Ready for PR?
- [x] I have updated the CHANGELOG with an entry including a description, a link to my profile, and a link to the issue ticket. This change is filed under an Enhancement or Bug Fix, is shown under the relevant release tag name, and is included in my PR branch.
- [x] I have run `lein run-lint` to check for linting errors.

## Changes Proposed:
- Adds a click indicator for buttons to make it more clear that the simulator is actually working.

## Screenshot of feature:
Very very hard to screenshot

@emilyseibert @oconn @dehli

### CODE REVIEW

**Comment by dehli** [comment]
This has conflicts that need to be resolved.

**Comment by elibosley** [comment]
Fixed 🔨

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ CHANGELOG
 
 **Bug Fixes**
 * Arena aspect ratio is set based on the arena size [Eli Bosley](/elibosley) #[345](https://github.com/willowtreeapps/wombats-web-client/issues/345)
-* Fixed issue with simualator pause button not working after window change [Eli Bosley][/eli] [#359](https://github.com/willowtreeapps/wombats-web-client/issues/359)
+* Fixed issue with simulator pause button not working after window change [Eli Bosley][/eli] [#359](https://github.com/willowtreeapps/wombats-web-client/issues/359)
```
Comment (dehli) [inline CHANGELOG.md]: Could you make these 3 bullets match the format that the rest of the bullets have later in the `CHANGELOG`?

**Comment by elibosley** [comment]
Alright, I think that that should be good to go!

**Pull request merged**